src/cdk/collections/selection.ts

```diff
@@ -34,7 +34,13 @@ export class SelectionModel<T> {
   }
 
   /** Event emitted when the value has changed. */
-  onChange: Subject<SelectionChange<T>> | null = this._emitChanges ? new Subject() : null;
+  changed: Subject<SelectionChange<T>> | null = this._emitChanges ? new Subject() : null;
+
+  /**
+   * Event emitted when the value has changed.
+   * @deprecated Use `changed` instead.
```

Comment:
Need to add a deletion-target here (8.0)

Reply:
Done :)